### PR TITLE
#169036299 - Configure clipped url to redirect to original url

### PR DESCRIPTION
**Pull request by rexdavinci**
Clipped url will be redirected to actual urls

### CODE REVIEW

File: src/controllers/link.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import Link from '../models/UrlShorten';
+module.exports = {
+  redirectLink: async (req, res)=>{
+    const {id} = req.params;
+    const fondLink = await Link.findOne({urlCode: id})
+    if(fondLink){
+      const actualLocation = foundLink.long_url
```
Comment (code2exe) [inline src/controllers/link.js:7]: I guess you meant fondLink.long_url  in line 7

Comment (rexdavinci) [reply src/controllers/link.js:7]: Ah yes. I will look into that. Thanks

Comment (code2exe) [reply src/controllers/link.js:7]: Did you test this before pushing?

Comment (rexdavinci) [reply src/controllers/link.js:7]: It will be fixed in a bit

**Review by shonubijerry — CHANGES_REQUESTED** [review]
Kindly rebase with the remote branch... Some merge were made that is not reflecting here

File: src/controllers/link.js (hunk 1 above)
Comment (shonubijerry) [reply src/controllers/link.js:7]: Hi @rexdavinci, has it been fixed?

**Pull request closed without merging**